lib/components/mp4muxer/index.ts

```diff
@@ -114,6 +166,38 @@ export class Mp4Muxer extends Tube {
       },
     })
 
+    const sendFtypMoovFromSdpMessage = (msg: SdpMessage): any => {
+      // Why is this here? These should be default inside the mvhd box?
+      const now = Math.floor(new Date().getTime() / 1000 + 2082852000)
```

Comment:
This appears to be an offset to make it seconds since 1904, would be good to have a comment here.

Reply:
This is moved code.